subgraphs/spotify/schema.graphql

```diff
@@ -1,3 +1,22 @@
+# @contact isn't being printed to the generated-schema.graphql file
+# See https://github.com/apollographql/federation/issues/1847
+directive @contact(
+  "Contact title of the subgraph owner"
+  name: String!
+  "URL where the subgraph's owner can be reached"
+  url: String
+  "Other relevant notes can be included here; supports markdown links"
+  description: String
+) on SCHEMA
+
+extend schema
+  @contact(
+    name: "Spotify Playback Subscription Server"
+    url: "https://myteam.slack.com/archives/teams-chat-room-url"
```

Comment:
Should these be filled out in a meaningful way?

Reply:
Yup, added pointers to our Discord